Array1D[] dim1
